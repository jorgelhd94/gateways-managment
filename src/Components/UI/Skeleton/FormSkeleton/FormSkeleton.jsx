import React from 'react';

const FormSkeleton = () => {
  return (
    <div className="bg-white rounded w-full mx-auto rounded-2xl mt-8">
      <div className="">
        <div className="grid grid-cols-3 gap-4 mt-2">
          <div className="h-16 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-16 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-16 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-16 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-16 bg-gray-200 rounded animate-pulse"></div>
          <div className="h-16 bg-gray-200 rounded animate-pulse"></div>
          </div>
      </div>
    </div>
  );
};

export default FormSkeleton;
