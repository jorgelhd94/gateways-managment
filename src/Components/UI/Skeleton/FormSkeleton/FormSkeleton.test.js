import React from 'react';
import renderer from 'react-test-renderer';
import FormSkeleton from './FormSkeleton';

test('FormSkeleton render correctly', () => {
  const component = renderer.create(<FormSkeleton />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
