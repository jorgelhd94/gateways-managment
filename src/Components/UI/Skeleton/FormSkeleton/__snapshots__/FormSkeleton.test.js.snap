// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSkeleton render correctly 1`] = `
<div
  className="bg-white w-full mx-auto rounded-2xl mt-8"
>
  <div
    className=""
  >
    <div
      className="grid grid-cols-3 gap-4 mt-2"
    >
      <div
        className="h-16 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-16 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-16 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-16 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-16 bg-gray-200 rounded animate-pulse"
      />
      <div
        className="h-16 bg-gray-200 rounded animate-pulse"
      />
    </div>
  </div>
</div>
`;
